release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1050,6 +1050,18 @@ The Special Resource Operator (SRO) includes a consistent log output format with
 
 Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.
 
+[id="ocp-4-11-RAN-disabling-chronyd"]
+====  Disable chronyd in the PolicyGenTemplate CR
+
+You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:
+
+[source,yaml]
+----
+[service]
+service.stalld=start,enable
```

Comment:
Seems like the line that is actually disabling `chronyd` is [this](https://github.com/openshift/openshift-docs/pull/48933/files#diff-c77c0cbc134fbd6d7bfb05509f65f4077fbbe88a6ce62ddafb44edc35e479f26R1062). Unless they are related, I suggest removing the following line. I also noticed that the following config is already mentioned in the released version.
```suggestion

```